src/gatsby/src/templates/parkSubPage.js

```diff
@@ -72,15 +72,27 @@ export default function ParkSubPage({ data }) {
   })
 
   const breadcrumbs = [
-    <Link key="1" href="/" underline="hover">
+    <GatsbyLink key="1" to="/">
       Home
-    </Link>,
-    <Link key="2" href="/find-a-park" underline="hover">
+    </GatsbyLink>,
+    <GatsbyLink
+      key="2"
+      to="/find-a-park"
+      onClick={(e) => {
+        if (sessionStorage.getItem("prevPath").includes('find-a-park')) {
```

Comment:
The first condition is never going to be true on a park subpage because find-a-park doesn't link to subpages.

you can just reduce it to one if statement....
```
if (sessionStorage.getItem("lastSearch")) {
          e.preventDefault();
          navigate('/find-a-park/' + sessionStorage.getItem("lastSearch"))
}
```